provisioning/transport/amqp/src/ProvisioningTransportHandlerAmqp.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Azure.Devices.Provisioning.Client.Transport
     /// </summary>
     public class ProvisioningTransportHandlerAmqp : ProvisioningTransportHandler
     {
-        private static readonly TimeSpan s_defaultOperationPoolingInterval = TimeSpan.FromSeconds(2);
+        private static readonly TimeSpan s_defaultOperationPollingInterval = TimeSpan.FromSeconds(2);
```

Comment:
A bit of a tangential change, but this is a **polling** interval not a **pooling** interval

Reply:
amqp just loves pooling everything